pkgs/applications/radio/direwolf/default.nix

```diff
@@ -22,22 +22,23 @@ stdenv.mkDerivation rec {
     espeak gpsd hamlib perl python3
   ] ++ (optionals stdenv.isLinux [alsa-lib udev]);
 
-  patches = [
-    ./udev-fix.patch
-  ];
-
   postPatch = ''
+    substituteInPlace conf/CMakeLists.txt \
+      --replace /etc/udev/rules.d/ $out/lib/udev/rules.d/
```

Comment:
Not sure if this should use placeholder.

Reply:
What do you mean by placeholder?

Reply:
``${placeholder "out"}``

Reply:
I realize this has now been merged, but I looked into it and it seems `placeholder` is only meant to be used in places where output paths are unavailable via environment variables (i.e. anywhere besides phase and pre-/post-phase scripts). Related forum post: https://discourse.nixos.org/t/what-is-the-difference-between-placeholder-out-and-out/4862/2